ddtrace/internal/sma.py

```diff
@@ -15,16 +15,12 @@ class SimpleMovingAverage(object):
         "sum_total",
     )
 
-    def __init__(self, size):
-        # type: (int) -> None
+    def __init__(self, size: int) -> None:
         """
-        Constructor for SimpleMovingAverage.
-
         :param size: The size of the window to calculate the moving average.
         :type size: :obj:`int`
         """
-        if size < 1:
-            raise ValueError
+        size = max(1, size)
```

Comment:
Would we ever want to know if size < 1? Or is it fine to just always set it to a minimum of 1 even if something weird is happening and never know about it?

Reply:
Given that the prior behavior was a crash, I don't really think so.